### PR TITLE
Updated responsive CardDef embedded format

### PR DESCRIPTION
**Pull request by habdelra**
This PR includes the finalized CardDef default embedded format design. The final design is designed around the aspect ratio of the space yielded to the card. The following dimensions were included in the design doc which our updated implementation should match now. The design is here https://app.zeplin.io/project/6682e084fe8b0edd27420fea/screen/669049068144f366593a7c54

Screenshot from 2024-07-17 07-30-07
Screenshot from 2024-07-17 07-30-42
Screenshot from 2024-07-17 07-30-32

### CODE REVIEW

File: packages/host/tests/integration/components/card-basics-test.gts [hunk 1]
```diff
@@ -590,75 +590,22 @@ module('Integration | card-basics', function (hooks) {
         .dom('[data-test-viewport="large"] [data-test-card-thumbnail-text]')
         .doesNotExist();
 
-      await percySnapshot(assert);
-    });
-
-    test('renders a default (CardDef) embedded view for card without thumbnail', async function (assert) {
```
Comment (habdelra) [inline packages/host/tests/integration/components/card-basics-test.gts:596]: removed this so we don't waste our percy budget

**Comment by backspace** [comment]
I’ll look at the code now but running locally I see a couple of visual flaws, like the bottom verticals at the corners being missing on the tiny card:

Boxel 2024-07-17 16-53-51

or a strange hover effect:

screencast 2024-07-17 16-54-53

Maybe these are preexisting problems though, or out of scope for this.

**Review by backspace — APPROVED** [review]
I checked the Percy snapshots too, looks good!

**Comment by habdelra** [comment]
> I’ll look at the code now but running locally I see a couple of visual flaws, like the bottom verticals at the corners being missing on the tiny card:
> 
> Boxel 2024-07-17 16-53-51
> or a strange hover effect:
> 
> screencast 2024-07-17 16-54-53 [ screencast 2024-07-17 16-54-53 ](https://private-user-images.githubusercontent.com/43280/349694828-f35de30c-a02d-43cf-9a4f-3e75010444ef.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyNjY2NzYsIm5iZiI6MTcyMTI2NjM3NiwicGF0aCI6Ii80MzI4MC8zNDk2OTQ4MjgtZjM1ZGUzMGMtYTAyZC00M2NmLTlhNGYtM2U3NTAxMDQ0NGVmLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE4VDAxMzI1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1ZGI2ZjliN2RjZDA4YWI2MzBhYzE1MDdiOTlkOWU2Njg1MWZiZmNkMTBmYTE0YzM3YjNjZjVjYzI0NmQ0MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tofjUWaQk-jD-wqzh8jC3_fMcv8AMOjAqR2h1RdWne4) [ ](https://private-user-images.githubusercontent.com/43280/349694828-f35de30c-a02d-43cf-9a4f-3e75010444ef.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyNjY2NzYsIm5iZiI6MTcyMTI2NjM3NiwicGF0aCI6Ii80MzI4MC8zNDk2OTQ4MjgtZjM1ZGUzMGMtYTAyZC00M2NmLTlhNGYtM2U3NTAxMDQ0NGVmLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE4VDAxMzI1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1ZGI2ZjliN2RjZDA4YWI2MzBhYzE1MDdiOTlkOWU2Njg1MWZiZmNkMTBmYTE0YzM3YjNjZjVjYzI0NmQ0MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tofjUWaQk-jD-wqzh8jC3_fMcv8AMOjAqR2h1RdWne4)
> 
> Maybe these are preexisting problems though, or out of scope for this.

Yeah, this is out of scope for this ticket. Specifically this ticket deals with the style within the card boundary. The items you noted are issues with the overlay which lives outside of the card boundary. I can make a new ticket in linear tho to address this.

**Review by jurgenwerk — APPROVED** [review]
Very responsive!

**Comment by habdelra** [comment]
I filed ticket https://linear.app/cardstack/issue/CS-7010/update-card-overlay-for-new-carddef-embedded-template-layout to update the overlay layout based on the new embedded card styles

**Pull request merged**